src/OpenTelemetry/Metric/MetricProvider.cs

```diff
@@ -0,0 +1,120 @@
+// <copyright file="MetricProvider.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+
+#nullable enable
+
+namespace OpenTelemetry.Metric

```

Comment:
Metrics instead of Metric? https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/library-layout.md#sdk-package